docs/config.md

```diff
@@ -1612,6 +1613,18 @@ Default: `false`
 
 Type: `flag`
 
+### `Routing.LoopbackAddressesOnLanDHT`
+
+**EXPERIMENTAL: `Routing.LoopbackAddressesOnLanDHT` configuration may change in future release**
+
+Whether loopback addresses (e.g. 127.0.0.1) should not be ignored on the local LAN DHT.
+
+Most users do not need this setting. It can be useful during testing, when multiple Kubo nodes run on the same machine but some of them do not have `Discovery.MDNS.Enabled`.
```

Comment:
:information_source: I played a bit with various setups and this seems to be useful only when mDNS discovery is not available. Had two nodes with MDNS, and they discovered each other and connected over `/ip6/::1/udp/4001/quic-v1/` just fine.

Added this note so our config docs explain why and when the option matters, so users save time and not bother with it.